### PR TITLE
docs: add inference refs (Torchserve, TGI Benchmark, Raycast)

### PR DESCRIPTION
**Pull request by SaboniAmine**
Add inference references : 
- TorchServe is a PyTorch official serving infrastructure, highly integrated in kubernetes-based environments, robust and scalable.
- TGI Benchmarks overviews the configuration of TGI for performance optimization, but this video is not easy to find on YouTube. This should help choosing and optimizing production setups.

Other : 
- [Raycast](https://www.raycast.com/). I couldn't find a fitting category for this one, but my guess is that could help developers used to ollama usage extending its capabilities to apps outside of the terminal. The initial app doesn't seem open source, but all community extensions are.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +105,7 @@ _Note_: The foundation model can be found under their associated task.
 ### Tools
 
 - [Text Generation Inference](https://huggingface.co/docs/text-generation-inference/index): Toolkit to serve large language models.
-- [Text Embedding Inference](https://huggingface.co/docs/text-embeddings-inference/index): Toolkit to serve text embeddings.
+- [TGI Benchmark](https://www.youtube.com/watch?v=jlMAX2Oaht0): Understanding throughput of inference servers, with TGI example.
```
Comment (merveenoyan) [inline README.md:109]: Why remove TEI here?

**Review by merveenoyan — COMMENTED** [review]
Thank you! ^_^

File: README.md (hunk 1 above)
Comment (SaboniAmine) [reply README.md:109]: ... I should not commit after midnight. I thought I duplicated it by inadvertance. Should be good now!

**Comment by SaboniAmine** [comment]
> Thank you! ^_^

Thanks for this repo! If it's okay, I might suggest additions from time to time.

**Pull request merged**